pkgs/by-name/ne/nezha/adminFrontend.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "nezha-admin-frontend";
+  version = "1.0.15";
+
+  src = fetchFromGitHub {
+    owner = "nezhahq";
+    repo = "admin-frontend";
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Could you please provide some references? As I understand, the `refs/tags/` path is used to limit the scope to the specific tag, but not the same name branch.

Reply:
https://ryantm.github.io/nixpkgs/builders/fetchers/#fetchfromgithub
> `rev` corresponds to the Git commit hash or tag (e.g v1.0) that will be downloaded from Git.